sklearn/compose/_column_transformer.py

```diff
@@ -543,14 +583,18 @@ def _update_fitted_transformers(self, transformers):
         # transformers are fitted; excludes 'drop' cases
         fitted_transformers = iter(transformers)
         transformers_ = []
+        self._name_to_fitted_passthrough = {}
 
         for name, old, column, _ in self._iter():
             if old == "drop":
                 trans = "drop"
             elif old == "passthrough":
                 # FunctionTransformer is present in list of transformers,
                 # so get next transformer, but save original string
```

Comment:
I think this line refer to `trans="passthrough"`. If both could be next, it would be better.